src/games/supported/Pong.cpp

```diff
@@ -96,3 +99,29 @@ void PongSettings::loadState(Deserializer & ser) {
   m_terminal = ser.getBool();
 }
 
+// returns a list of mode that the game can be played in
+ModeVect PongSettings::getAvailableModes(){
```

Comment:
There are also four difficulties (two settings for left and two for right).

Also, we've been doing Pong wrong all along. The manual says you can use the fire button to speed up the ball (in both modes). The fire button's behaviour is actually the only difference between the two modes.